apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/BaseChatView.java

```diff
@@ -215,4 +248,14 @@ void updateSelection(boolean isSelected) {
             getContent().pseudoClassStateChanged(SELECTED_PSEUDO_CLASS, isSelected);
         }
     }
+
+    private static class IconIdPair {
```

Comment:
As IconIdPair is only used inside the class we could use the Pair<> class as well. But no need to change...